# A message to contain all the info about the detected humans in each scan
Header header
ClusteredScan[] clusterData
